modules/storage-persistent-storage-pv.adoc

```diff
@@ -126,7 +126,7 @@ The following table lists the access modes:
 |ReadWriteOnce
 |`RWO`
 |The volume can be mounted as read-write by a single node.
-|ReadWriteOncePod ^[1]^
+|ReadWriteOncePod
```

Comment:
Do we want to add selinux context mount here as a note?

Reply:
Yes. Done.